coala_quickstart/green_mode/green_mode.py

```diff
@@ -339,13 +350,12 @@ def run_test_on_each_bear(bear, file_dict, file_names, lang, kwargs,
     if type_of_setting == 'non-op':
         printer.print('Finding suitable values to necessary '
                       'settings for ' + bear.__name__ +
-                      ' based on your project ...',
-                      color='green')
+                      ' based on your project ...')
     else:
         printer.print('Finding suitable values to all settings '
                       'for ' + bear.__name__ +
                       ' based on your project ...',
-                      color='yellow'
+                      color='green'
```

Comment:
Separate commit for this and https://github.com/coala/coala-quickstart/pull/292/commits/e57c99ee9cbfea29d6dc6b15e7aa0a4012604072#diff-fc8b14e8f3fc31225649bf250ebd04c9R353